shared/chat/conversation/info-panel/settings/index.tsx

```diff
@@ -177,11 +270,22 @@ const styles = Styles.styleSheetCreate(
         },
         isTablet: {alignSelf: 'center', maxWidth: 600},
       }),
+      settingsHeader: {
+        marginBottom: Styles.globalMargins.small,
+        paddingLeft: Styles.globalMargins.small,
```

Comment:
if you want to get multiple paddings in one, you can use `Styles.padding` which lets you pass less than 4 paddings but get all 4, like the `padding: 0px 5px` would in real CSS